Dockerfile

```diff
@@ -1,16 +1,17 @@
+# Build SELinux packages for Arch linux in Docker/Podman container
+
 # Usage:
-#    sudo docker build -t arch-selinux-build .
-#
-# or with Podman on Arch Linux in July 2020:
-#    podman --cgroup-manager=cgroupfs build -t arch-selinux-build .
+#    docker build -t arch-selinux-build .
+# or   
+#    sudo podman build -t arch-selinux-build .
 #
-# Once the container is build, you can get the packages in "pkgs" directory with:
-#    sudo docker run -v "$(pwd)/pkgs:/packages" --rm -ti arch-selinux-build
+# Once the container is built, you can get the packages in "pkgs" directory with:
+#    docker run -v "$(pwd)/pkgs:/packages" --rm -ti arch-selinux-build
 # or
 #    podman run -v "$(pwd)/pkgs:/packages" --rm -ti arch-selinux-build
 
-# Use official Arch Linux Docker image (https://hub.docker.com/_/archlinux)
-FROM archlinux:latest
+# Use official Arch Linux Docker image (https://hub.docker.com/r/archlinux/archlinux)
+FROM docker.io/archlinux/archlinux:latest
```

Comment:
https://github.com/archlinux/archlinux-docker#arch-linux-docker-image is not clear at all about what is the official image, and both images were updated recently (and actually `archlinux` *is* more recent than `archlinux/archlinux`, now):
```console
$ sudo docker images | grep archlinux
archlinux                      latest              ddcfcc25b371        12 hours ago        585MB
archlinux/archlinux            latest              f1713b6d04c3        23 hours ago        472MB
```

Anyway, the `archlinux/archlinux` looks "more official" so I agree on switching to it, and :+1: with adding `docker.io/`.

Reply:
Here is the official explanation of the differences with the images, I put this link into the comment section.
https://gitlab.archlinux.org/archlinux/archlinux-docker#pipeline